contents/blog/the-posthog-array-1-16-0.md

```diff
@@ -0,0 +1,202 @@
+---
+date: 2020-11-04
+title: Array 1.16.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Following our largest release to date, we are now back on a more regular release schedule. And, given that scalability was the focus of the previous release, we thought it was time for some fresh new features. 
+
+So, without further ado, here they are:
+
+## Release Notes
+
+### [Session Recording (Beta)](https://github.com/PostHog/posthog/pulls?q=is%3Apr+session+recording+)
+
+![Session Recording Page Screenshot](../images/blog/array/session-recording.png)
+
+Given that our stated mission at PostHog is to increase the number of successful projects in the world, session recording felt like a feature that fits in perfectly with that goal.
+
+PostHog already provides various features to help you understand and improve your UX - but watching real users use your product is a _whole nother ball game_. 
+
+With PostHog's session recording, you are able to truly feel the pain point of your users first-hand, seeing where they get stuck, debugging exceptions faster, and making your UX smoother. 
```

Comment:
"pain points" or "where the pain points are" maybe?

Reply:
You could also lean in a bit into the positive side, like "delight moments"